db/column_family_test.cc

```diff
@@ -280,7 +282,8 @@ class ColumnFamilyTestBase : public testing::Test {
       ColumnFamilyDescriptor desc;
       ASSERT_OK(handles_[cfi]->GetDescriptor(&desc));
       RocksDBOptionsParser::VerifyCFOptions(ConfigOptions(), desc.options,
-                                            current_cf_opt);
+                                            current_cf_opt)
+          .PermitUncheckedError();
```

Comment:
Why is this status being ignored?  After spending a lot of time in the Options code, I am especially curious about this one...

Reply:
In the test itself it expects the arguments to be invalid so it returns invalid in few cases and OK in other.

Reply:
@mrambacher, Oh my bad. This is different. It gives an error that 
"RocksDBOptionsParser::VerifyCFOptions( ConfigOptions(), desc.options, current_cf_opt)
Invalid argument: [RocksDBOptionsParser]: failed the verification on ColumnFamilyOptions::periodic_compaction_seconds--- The specified one is 0 while the persisted one is 18446744073709551614."
Line: https://github.com/facebook/rocksdb/blob/7d472accdca996d7d83ae8ce78ad17f799696926/options/options_parser.cc#L679

Reply:
I am not sure if this is expected or not.

Reply:
I do not think that is expected.  I will take a quick look...

Reply:
The problem is that when you add a ColumnFamily to the DB, the ColumnFamilyOptions get sanitized and some of the default values might change.  If you change the line to this:
      // Need to sanitize the default column family options before comparing them...
      ASSERT_OK(RocksDBOptionsParser::VerifyCFOptions(ConfigOptions(), desc.options,
                                                      SanitizeOptions(dbfull()->immutable_db_options(), current_cf_opt)));

The tests should pass.

Reply:
Yes that works. Thanks a lot!